### PR TITLE
Code files,badges all are present in separate folders

### LINKED ISSUE #37: Uploads folder contain code files and created badges folders
**I'm submitting a ...**  (check one with "x")
- [x ] feature request
**Current behavior:**
 Uploads folder contains code files and created badges folder, which should not happen .

**Expected behavior:**
Uploads should only contain uploaded files and code files should be present in app/

### PR DESCRIPTION
**Pull request by nanspro**
Fixes #37 

#### Short description of what this resolves:
-Code is separated from static files.
-Proper directories arrangement

#### Changes proposed in this pull request:
-Uploads folder now only contains uploaded csv and png files.
-static/badges contain badges formed by uploaded files and also contains all-badges.pdf
-generate-badges.py and merge_badges.sh are now present in app/.

workspace 1_009

### CODE REVIEW

**Comment by nanspro** [comment]
@jajodiaraghav  i have implemented changes you asked

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ If you like to join developing,
 - you can find/create [issues](https://github.com/fossasia/badgeyay/issues) and solve them.
   - When you solve an issue, you do not own it. Share your progress via a Pull-Requst as soon as possible.
   - Discuss with others who work on the issue about the best solution. It is your responsibility, not the maintainer's to choose the best solution.
-- If in doubt, let's follow [CCCC][cccc].
+- If in doubt, let's follow [CCCC](https://rfc.zeromq.org/spec:42/C4).
```
Comment (jajodiaraghav) [inline README.md]: I think you forgot to rebase!

File: README.md [hunk 2]
```diff
@@ -74,26 +74,8 @@ them together into one.
...
-
-
-[gitter]: https://gitter.im/fossasia/badgeyay
-[cccc]: https://rfc.zeromq.org/spec:42/C4
-[team]: https://github.com/orgs/fossasia/teams/badgeyay-admin/members
+The project is maintained by
+ - Nicco Kunzmann ([@niccokunzmann](https://github.com/niccokunzmann))
+ - Tarun Kumar ([@meets2tarun](https://github.com/meets2tarun))
+ 
+ [gitter]: https://gitter.im/fossasia/open-event-orga-server
```
Comment (jajodiaraghav) [inline README.md]: I think you forgot to rebase!

File: app/main.py [hunk 3]
```diff
@@ -16,8 +16,8 @@ def index():
 	return render_template('index.html')
 
 def generate_badges():
-    os.system(os.path.join(app.config['UPLOAD_FOLDER'], 'merge_badges.sh'))
-
+    os.system(SCRIPT)
+
```
Comment (jajodiaraghav) [inline app/main.py]: Why is this change needed?

**Comment by meets2tarun** [comment]
@nanspro 
Kindly rebase!

**Comment by nanspro** [comment]
Should i create a pull request again after changing readme.md

File: app/main.py (hunk 3 above)
Comment (nanspro) [reply app/main.py]: codacy was showing error in that previously

**Comment by jajodiaraghav** [comment]
Why did you delete the camera and guest PNG images? And why did you add a PDF file in fonts folder?

**Comment by meets2tarun** [comment]
> Should i create a pull request again after changing readme.md

I would suggest you to follow on this PR.

**Comment by nanspro** [comment]
i was checking my code on these two files because i think it should contain uploaded images and csv only.

**Comment by jajodiaraghav** [comment]
@nanspro Please do not stack commits one after the other in a single PR. Please squash your commits into a single commit [ A single task should reside in a single commit ]
https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by nanspro** [comment]
ok i will make another PR

**Comment by meets2tarun** [comment]
This arises the problem of not following the basics of a good PR.
One should solve atmost one issue in a PR and make a PR from a different branch of the forked repo.
And kindly rebase, clear conflicts if arises rather than closing the PR.

**Comment by jajodiaraghav** [comment]
I think @nanspro  is new to Open Source. I would suggest you to read a little about git. It is impossible to contribute to FOSS unless one has a clear understanding of VCS.

Feel free to ask for advice. Remember, we are here to help :)
